analysis/create_stamps.py

```diff
@@ -51,34 +78,69 @@ def load_psi_phi(self, psi_filename, phi_filename, lc_index_filename):
         return(psi, phi, lc_index)
 
     def load_times(self, time_filename):
+        """
+        Load the image time stamps.
+    
+        Arguments:
+            time_filename - The filename of the time data.
 
+        Returns:
+            times - A list of times for each image.
+        """
         times = []
         with open(time_filename, 'r') as f:
             reader = csv.reader(f)
             for row in reader:
                 times.append(np.array(row, dtype=np.float))
-
         return times
 
     def load_stamps(self, stamp_filename):
+        """
+        Load the stamps.
+    
+        Arguments:
+            stamp_filename - The filename of the stamp data.
 
+        Returns:
+            stamps - A list of np.arrays containing the stamps
+                     for each result.
+        """
         stamps = np.genfromtxt(stamp_filename)
         if len(np.shape(stamps)) < 2:
             stamps = np.array([stamps])
-        #stamp_normalized = stamps/np.sum(stamps, axis=1).reshape(len(stamps), 1)
 
         return stamps
 
     def stamp_filter(self, stamps, center_thresh, verbose=True):
-
+        """
```

Comment:
Maybe we want to rename this function to something more specific since there are different stages of "stamp filtering"?

Reply:
Good call. I updated this to max_value_stamp_filter